packages/state-transition/src/util/validator.ts

```diff
@@ -67,6 +67,14 @@ export function getBalanceChurnLimit(epochCtx: EpochCache): number {
 }
 
 export function getActivationExitChurnLimit(epochCtx: EpochCache): number {
+  /**
+   * This is a pseudo-fork to support the "hack" option D discussed here:
+   * https://notes.ethereum.org/@ethpandaops/post-finality-path-forward-holesky
+   */
+  const HOLESKY_REVIVAL_EPOCH = Infinity;
+  if (epochCtx.epoch >= HOLESKY_REVIVAL_EPOCH) {
+    return epochCtx.config.MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT;
```

Comment:
this would probably be another constant or just literally hard coding a number here, in theory `HOLESKY_REVIVAL_EPOCH` could even be hard coded if we use a separate branch